apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.html

```diff
@@ -314,4 +314,4 @@ <h2 class="header">
     }
   </div>
 </form>
-<cde-footer></cde-footer>
+<hra-footer logo="assets/logo/hra_logo.svg"></hra-footer>
```

Comment:
` logo="assets/logo/hra_logo.svg"` -- make `assets/logo/hra_logo.svg` be the default logo URL and remove from usage here and in dashboard.

Reply:
modified